config/dev.env

```diff
@@ -21,3 +21,4 @@ SESSION_SECRET='aria_at_report_dev'
 AUTOMATION_SCHEDULER_URL=http://localhost:8866
 AUTOMATION_SCHEDULER_SECRET=3Bq10QMDcca2IuxrTr23!eAsDCoILhHrrlBda7-3-WE9?El2S?R5REWXTyE3B72!
 AUTOMATION_SCHEDULER_PORT=8866
+AUTOMATION_CALLBACK_FQDN=
```

Comment:
Can we use `API_SERVER` for this?

Reply:
Makes sense to me!

Reply:
Was also realizing while looking at the other configuration settings, on the `dev` config, API_SERVER is set to `localhost:8000` - this really needs to be an FQDN that the github action can POST to, and may or may not be equal to the `API_SERVER` value